src/lib/select/select.spec.ts

```diff
@@ -613,6 +613,33 @@ describe('MatSelect', () => {
         expect(() => fixture.componentInstance.select.open()).not.toThrow();
       });
 
+      it('should set the width of the overlay based on the trigger and resize', fakeAsync(() => {
+        trigger.style.width = '200px';
+        fixture.detectChanges();
+        fixture.whenStable().then(() => {
```

Comment:
You can't use `fixture.whenStable` inside a `fakeAsync` test. What ends up happening is that the test will complete before the callback is invoked. What I meant before is taking something like this:

```ts
fixture.detectChanges();
fixture.whenStable(() => {
  // do stuff
});
```

And turning it into this:

```ts
fixture.detectChanges();
flush();
// do stuff
```

Reply:
I updated that already, missed it before haha

Reply:
It seems like you have one left over.

Reply:
oh snap, my bad... thats what i get for trying to do it fast haha.. fixing it

Reply:
now its done for sure! 😄